data/src/main/java/co/nimblehq/data/service/providers/MoshiBuilderProvider.kt

```diff
@@ -0,0 +1,16 @@
+package co.nimblehq.data.service.providers
+
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.adapters.Rfc3339DateJsonAdapter
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import java.util.*
+
+object MoshiBuilderProvider {
+
+    val moshiBuilder: Moshi.Builder
+        get() {
+            return Moshi.Builder()
```

Comment:
we could get rid of return by `get() = Moshi.Builder()...`

Reply:
nice refactor, let me update 👍